.env.example

```diff
@@ -1,4 +1,4 @@
-STEP_ROOT='tmp/CA/root_ca.crt'
+STEP_ROOT='tmp/secrets/CA/root_ca.crt'
 STEP_CA_URL='https://localhost:55555'
```

Comment:
I don't think we need any CA env variables for agent. The step-ca stuff is only for experimenting with CA features, we are meant to create our own PKI eventually. So embedding the env variables here shouldn't be necessary. I suppose you could put it in your own local `.env`. And update the README.md for what should be done if we want to test some PKI feature.

Reply:
sounds good, I will leave it in as an option and tell the user in README that if they want to use step-cli for testing, they will needs these environment variables

Reply:
Cool